### ISSUE TITLE
[to add] ssl.googleanalytics.com

### ISSUE THREAD

**Issue by dnmTX** [lightswitch05/hosts#287]
Wonder why is this not blocked like in every hosts list out there already? 

Also found this:  `pixel.admobclick.com`
![tmar](https://user-images.githubusercontent.com/34774426/122455778-63a65a80-cf72-11eb-8d26-058d388bc0bf.png)

@llacb47 you have any input on either one?

**Comment by lightswitch05** [comment]
well, the reason they are not blocked in this project is because I've never seen them before. You've given me enough evidence to add `pixel.admobclick.com` and `admobclick.com`. But I've never seen `ssl.googleanalytics.com` used anywhere before.

**Comment by dnmTX** [comment]
>  But I've never seen ssl.googleanalytics.com used anywhere before.
I actually did see it(in AdGuard's log),but never paid attention cause it was blocked by it. 
When some `googleanalytics.com` domain gets blocked initially it tries this one as a alternative.

**Comment by dnmTX** [comment]
Check this out:
Hosted on the same IP:
![sameIP](https://user-images.githubusercontent.com/34774426/122458457-70787d80-cf75-11eb-91aa-323505cc4022.png)

And there are even more:
![siblings](https://user-images.githubusercontent.com/34774426/122458506-7d956c80-cf75-11eb-9002-8acde8dd3f43.png)

**Comment by lightswitch05** [comment]
they will both be added in the next list update 👍

**Comment by dnmTX** [comment]
What about the siblings? They do look suspicious as hell though.

**Comment by lightswitch05** [comment]
I added the root domains for them both - the list will auto-expand them

**Comment by dnmTX** [comment]
I see,good thinking 👍  
Dan,you don't mind call Stevo here to add those to his list instead of opening another issue there?

**Comment by lightswitch05** [comment]
Steven is more then welcome here, I lurk in his repo all the time

**Comment by dnmTX** [comment]
**ping** Steve @StevenBlack,if you don't mind add those to your list please. Thank you 👍 

0.0.0.0 ssl.googleanalytics.com
0.0.0.0 click.googleanalytics.com
0.0.0.0 alt2.googleanalytics.com
0.0.0.0 alt4.googleanalytics.com
0.0.0.0 sandbox.googleanalytics.com
0.0.0.0 admobclick.com
0.0.0.0 www.admobclick.com
0.0.0.0 pixel.admobclick.com

**Comment by StevenBlack** [comment]
Thanks Dan 'n Dan @lightswitch05 @dnmTX, done!

**Comment by dnmTX** [comment]
> Thanks Dan 'n Dan @lightswitch05 @dnmTX, done!

No,no....THANK YOU 👉 
😉

**Comment by dnmTX** [comment]
Dan @lightswitch05,note this first,it's work in progress,needs more input !
Also found `*.advanced-web-analytics.com` in `bankofamerica.com`'s website out of all places. 
It's implemented as `iframe` (don't laugh,this was way before even thinking of userscripts 😄 ) but blocking:
`rail.bankofamerica.com` ,which was the delivery domain,fixed it.
Just to note,there are like thousands of `advanced-web-analytics.com` variations,and your list is like,huge enough already.

**Comment by dnmTX** [comment]
`*.eloqua.com`

![PitchBook](https://user-images.githubusercontent.com/34774426/122478324-d291ac80-cf8e-11eb-8a0e-f498a02adef0.png)
